llvm/lib/Target/RISCV/RISCVSystemOperands.td

```diff
@@ -239,6 +240,7 @@ def : SysReg<"mbadaddr", 0x343>;
 def : SysReg<"mip", 0x344>;
 def : SysReg<"mtinst", 0x34A>;
 def : SysReg<"mtval2", 0x34B>;
+def : SysReg<"medelegh", 0x312>;
```

Comment:
Need `let isRV32Only = 1 in`

Should this be in the same group with "medeleg"?